### DISCUSSION TITLE
Uso de librerías extra

### DISCUSSION THREAD

**Discussion by DavidZang17** [IIC2133-PUC/2024-2#165] [Tarea 2] [answered]
Me preguntaba si se pueden usar las librerias math.h y limits.h, vi que preguntaron por math.h en otra discussion, pero era especifico a uso de sqrt()

**Comment by Segismundios** [comment]
Hola @DavidZang17.  La librería math.h la puedes usar sin problema. Por otro lado, para qué quieres usar limits.h?

**Reply by DavidZang17** [reply]
Buenisimo lo de math.h, en cuanto a limits.h, lo habia usado para tener acceso a INT_MAX e INT_MIN. pero creo que lo puedo sacar y usar otra cosa para hacer comparaciones entre ID

**Reply by tomastrivino** [reply]
AAAH pero creo que puedes acceder a INT_MAX, INT_MIN sin importar limits.h

**Reply by DavidZang17** [reply]
Creo que si, voy a limpiar el codigo y creo que no lo deberia necesitar

**Reply by DavidZang17** [reply]
Me tira error en make cuando saco el limits.h y uso el INT_MAX e INT_MIN. Puedo usarlo entonces?

**Reply by DavidZang17** [reply]
Mas especificamente, lo estoy usando por si un nodo es NULL al recibir el max y min weight. Puedo usarlo entonces o debo buscar otra manera de resolver eso?

**Reply by tomastrivino** [reply, marked as answer]
Sí se puede usar limits.h, saludos!
